doc/Command-Reference.md

```diff
@@ -5121,6 +5121,37 @@ This command is to display the link-training status of the selected interfaces.
     Ethernet8      trained          on      up       up
   ```
 
+**show interfaces errors**
+
+The show interface errors command provides detailed statistics and error counters for MAC-level operations on an interface. It displays the status of various operational parameters, error counts, and timestamps for when these errors occurred.
+
+- Usage:
+  ```
+  show interfaces errors [<interface_name>]
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show interfaces errors Ethernet4
```

Comment:
we should have an additional column to show the current status. Last timestamp(UTC) column is not clear to say if this is last raised or last cleared time stamp. It would be good to have both.

Reply:
@qinchuanares we dont have the capability to show current status

Reply:
@qinchuanares there is no clear time. All the port errors lead to link down. So, we capture here only the SET time which coincide with port link down event.